internal/loader/loader.go

```diff
@@ -164,6 +164,23 @@ func normalizePackage(opts *config.IndexOpts, pkg *packages.Package) *packages.P
 	//		Path string = "github.com/efritz/pentimento"
 	//		Version string = "v0.0.0-20190429011147-ade47d831101"
 
+	if opts.IsGoPackagesDriverSet {
```

Comment:
```suggestion
	// As of golang.org/x/tools v0.26, the JSON object for representing packages
	// does not have a Module field:
	//   https://github.com/golang/tools/blob/v0.26.0/go/packages/packages.go#L609-L627
	// which means that cross-repo navigation may not work out-of-the-box
	// for 3rd party build systems like Bazel, Buck2, Please etc. using GOPACKAGESDRIVER
	if opts.IsGoPackagesDriverSet {
```

Reply:
Thanks, I applied the code suggestion. 

> There is also a code path initializing defaultGoVersion which needs a Go stdlib version for cross-repo navigation. Given that supporting cross-repo navigation with a custom driver is not a goal here, it's fine to use a placeholder value here with an explicit TODO.

I guess we can use defaultGoVersion for now, if someone needs to have a specific value here, they can use `--go-version` flag